app/assets/stylesheets/application.scss

```diff
@@ -22,9 +22,11 @@ main {
   -webkit-font-smoothing: antialiased;
 }
 
-p {
-  margin-top: pem(5, 16);
-  margin-bottom: pem(20, 16);
+.prototype {
```

Comment:
It might be nice to break prototype-specific things out into a separate Sass file. That way it'll be way easier to update this file the next time we have to match production.